docs/contributing.md

```diff
@@ -452,9 +451,9 @@ Installing the pre-commit hooks to git
 1. precommit-check.sh - sanity checks before committing code
 1. createsuperuser.sh - creates a default superuser (assumes debian env.)
 
-#### Submit a bug
+# B. Submitting Bugs for Third Party npm packages
```

Comment:
`# B. Submitting Bugs for Third Party npm Packages`

Reply:
Done